package.json

```diff
@@ -20,6 +19,7 @@
     "@types/react": "^18.0.14",
     "@types/react-dom": "^18.0.5",
     "buffer": "^6.0.3",
+    "ci": "^2.2.0",
```

Comment:
did you do by mistake `npm install ci`  instead of `npm ci` ?
can you do `npm rm ci` and check this dependecy is removed